### ISSUE TITLE
read_svg Error in viewbox[[4]] : subscript out of bounds

### ISSUE THREAD

**Issue by durraniu** [coolbutuseless/svgparser#3]
Thank you for creating `svgparser`. I am trying to import a svg file but getting an error. I am on Windows 10.

## Svg file
I have the [svg file here](https://uwin365-my.sharepoint.com/:u:/g/personal/durraniu_uwindsor_ca/ETeoMBTBwNxIlu5Aqc4FIxcBmdnChi5eMVGHlhi3jvscHQ?e=1RwHKF) if you want to download it.  

## Code to import svg and the error
I have installed the css parser and the `svgparser` packages. Following is what I tried with the error:

```
library(svgparser)

df <- svgparser::read_svg("road_network.svg", obj_type = 'data.frame')
Error in viewbox[[4]] : subscript out of bounds
In addition: Warning message:
In svgparser::read_svg("road_network.svg", obj_type = "data.frame") :
  NAs introduced by coercion
```
Please let me know what am I missing

**Comment by coolbutuseless** [comment]
Fixed this in v0.1.2

The issue was that I had assumed `viewBox` definition could only be whitespace delimited, but it turns out the numbers can also be separated by commas.  Fixed the splitting code.

**Comment by durraniu** [comment]
Thanks a lot. It works now.
